presto-main/src/main/java/io/prestosql/cost/CostCalculatorUsingExchanges.java

```diff
@@ -302,6 +303,13 @@ public PlanCostEstimate visitUnion(UnionNode node, Void context)
             return costForStreaming(node, LocalCostEstimate.zero());
         }
 
+        @Override
+        public PlanCostEstimate visitSample(SampleNode node, Void context)
+        {
+            LocalCostEstimate localCost = LocalCostEstimate.ofCpu(getStats(node).getOutputRowCount());
+            return costForSource(node, localCost);
```

Comment:
`costForSource` could perhaps be named "cost for leaf". It assumes (and, I think, enforces) that node has no sources, which is not the case for SampleNode

Reply:
Thanks. But I'm still not sure which to use `costForStreaming` and `costForAccumulation`.  I guess `costForAccumulation` is an estimation for aggregation type operator so we can use `costForStreamin`.